### ISSUE TITLE
[Issue]: Consider aspect ratio when calculating video resolution description

### ISSUE THREAD

**Issue by SenorSmartyPants** [jellyfin/jellyfin#6950]
### Please describe your bug

For a non 16:9 TV video, video resolution description does not accurately reflect the quality of the image.
A 1248x520 video is label 480p just same as a 720x480 video is. There is no difference noted without going into media info.

**Example**: 
media info
![image](https://user-images.githubusercontent.com/991618/144910464-9ed7d934-a389-488a-9c2f-44b58303610a.png)

Note the title is **480p**, but the video width is 1248 (almost 720p standard width of 1280) and aspect ratio is 2.40:1.  If title was based on width to figure out height, 720p would be selected. Or if height was normalized to a 16:9 aspect ration height would be closer 720.

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS:
- Virtualization: LSIO Docker
- Clients: browser
- Browser: Chrome 96
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by crobibero** [comment]
I think this is a client only display, I think the server just returns the dimensions and the client can use them as they wish.. someone from @jellyfin/web can confirm

**Comment by SenorSmartyPants** [comment]
Server API calculates the title field as well.

**Comment by dmitrylyzo** [comment]
**Title** is `DisplayTitle` of the stream - it comes from the server.
https://github.com/jellyfin/jellyfin-web/blob/60495dd16a0c5071e3c987b094b44ca221cc85bf/src/components/itemMediaInfo/itemMediaInfo.js#L74-L76

**Comment by Bond-009** [comment]
Should already be fixed by #6256

(It's in the alpha releases, you can test it)
